ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/CollectionSelection.tsx

```diff
@@ -167,24 +177,32 @@ function CollectionSelection({
                                 </SelectOption>
                             ))}
                         </Select>
-                    </FormLabelGroup>
-                </FlexItem>
-                {allowCreate && (
-                    <FlexItem>
+                    </FlexItem>
+                    <FlexItem spacer={{ default: 'spacerMd' }}>
                         <Button
-                            className="pf-u-mb-md"
-                            variant={ButtonVariant.secondary}
-                            onClick={onToggleCollectionModal}
+                            variant={ButtonVariant.tertiary}
+                            onClick={onOpenViewCollectionModal}
+                            isDisabled={selectedScopeId === ''}
                         >
-                            Create collection
+                            View
                         </Button>
                     </FlexItem>
-                )}
-            </Flex>
+                    {allowCreate && (
+                        <FlexItem>
+                            <Button
+                                variant={ButtonVariant.secondary}
+                                onClick={onOpenCreateCollectionModal}
+                            >
+                                Create collection
+                            </Button>
+                        </FlexItem>
+                    )}
+                </Flex>
+            </FormLabelGroup>
             {isCollectionModalOpen && (
                 <CollectionsFormModal
                     hasWriteAccessForCollections={allowCreate}
-                    modalAction={{ type: 'create' }}
+                    modalAction={modalAction}
```

Comment:
Reusing the same `CollectionFormModal`. Changing the `modalAction` will make sure to determine whether to show the `view` modal vs. the `create` modal